hack/update-codegen.sh

```diff
@@ -17,8 +17,8 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
-CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../code-generator)}
-${CODEGEN_PKG}/generate-groups.sh "deepcopy,defaulter,client,lister,informer" \
+SCRIPT_ROOT="$(dirname "${BASH_SOURCE[0]}")/.."
```

Comment:
most of the test-infra scripts use `$(git rev-parse --show-toplevel)` which has some up and downsides. we should maybe standardize on one of these /shrug

someone should write a good, portable bash library to get the path to the running script :^)